runtime/src/main/java/org/corfudb/runtime/clients/ManagementClient.java

```diff
@@ -102,4 +114,20 @@ private static Object handleAlreadyBootstrappedError(CorfuMsg msg,
     public CompletableFuture<byte[]> sendHeartbeatRequest() {
         return router.sendMessageAndGetCompletable(CorfuMsgType.HEARTBEAT_REQUEST.msg());
     }
+
+    public AddNodeResponse addNodeRequest(String endpoint) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)